### DISCUSSION TITLE
Where to discuss GitHub Projects?

### DISCUSSION THREAD

**Discussion by abalter** [community/community#52839] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Unless I'm missing it, I don't see a slot under Categories > GitHub Product Categories for Projects.

Where is the best place to discuss Projects?

In particular to give feedback about the new beta Projects.

**Comment by CodeMaster7000** [comment, marked as answer]
Hi @abalter,

The best place to put feedback for Projects is [here](https://github.com/orgs/community/discussions/new?category=issues).
